scripts/globals/bluemagic.lua

```diff
@@ -263,37 +105,76 @@ function BluePhysicalSpell(caster, target, spell, params)
 
     -- print("Final D is ".. finalD)
 
-    -----------------------------------
-    -- Get the possible pDIF range and hit rate
-    -----------------------------------
-    if params.offcratiomod == nil then -- default to attack. Pretty much every physical spell will use this, Cannonball being the exception.
-        params.offcratiomod = caster:getStat(xi.mod.ATT)
+    ----------------------------------------------
+    -- Get the possible pDIF range and hit rate --
+    ----------------------------------------------
+    if (params.attkbonus == nil) then
```

Comment:
Just some minor styling, but parens are not necessary here, or any case outside of an and/or grouping.

Reply:
When I was testing it returned as "nil" and errors unless I put it in every lua. This allows you to not have to add it into every lua with a value since attkbonus isn't used in every single spell.

Or am I wrong?

Reply:
Parenthesis should have no impact on the returned value.  That's what I'm referring to.